raiden/tests/integration/cli/conftest.py

```diff
@@ -70,7 +70,16 @@ def changed_args():
 
 
 @pytest.fixture()
-def cli_args(request, tmpdir, raiden_testchain, removed_args, changed_args, environment_type):
+def cli_args(
+    request,
+    tmpdir,
+    raiden_testchain,
+    removed_args,
+    changed_args,
+    environment_type,
+    cleanup_tasks,  # pylint: disable=unused-argument
```

Comment:
`cleanup_tasks` should be run last, if a fixture itself uses greenlets. This is the least hacky way I found to order fixtures.